cfme/tests/infrastructure/test_host.py

```diff
@@ -617,3 +617,22 @@ def test_infrastructure_hosts_crud(appliance, setup_provider):
     # Case6 - lastly do the delete. First try is canceled.
     host.delete(cancel=True)
     host.delete
+
+
+@test_requirements.infra_hosts
+def test_infrastructure_hosts_tagging(appliance, setup_provider):
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
```

Comment:
Since this is a new test should `caseimportance` be `high`? Is this automating an existing manual test case?

Reply:
Good catch John. I copied this over from another test and didn't update correctly. The manual test this is automating is set to High importance. Setting this to High.

Reply:
@prichard77 if the manual case still exists, we should remove it in this PR too.

Reply:
Of course. I need to get into the habit of removing the existing manual test in the same PR as the work to automate the test.